code/modules/research/techweb/all_nodes.dm

```diff
@@ -1012,7 +1012,7 @@
 	/obj/item/weldingtool/abductor, /obj/item/wirecutters/abductor, /obj/item/circuitboard/machine/abductor, /obj/item/abductor/baton, /obj/item/abductor)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
 	hidden = TRUE
-	design_ids = list("alienalloy")
+	design_ids = list("alienalloy", "mind_flayer")
```

Comment:
should be illegal tech too considering the lethality

Reply:
Alien tech is even rarer than illegal, there’s hundreds of syndie toolboxes, there’s way less abductor shit around.